cmd/kubeadm/app/apis/kubeadm/v1beta1/defaults.go

```diff
@@ -101,6 +101,12 @@ func SetDefaults_ClusterConfiguration(obj *ClusterConfiguration) {
 	SetDefaults_DNS(obj)
 	SetDefaults_Etcd(obj)
 	SetDefaults_APIServer(&obj.APIServer)
+	SetDefaults_Scheduler(&obj.Scheduler)
+}
+
+func SetDefaults_Scheduler(obj *ControlPlaneComponent) {
+	obj.UID = 2000
+	obj.GID = 2000
```

Comment:
existing kubeadm APIs are locked - v1beta1, v2beta2 and new fields will not be added to them.
all new fields must be added to v1beta3.